### DISCUSSION TITLE
Add more sort settings.

### DISCUSSION THREAD

**Discussion by adumith** [microsoft-search/pnp-modern-search#3579] [Q&A]
Hello again,

On my results page, I've added the option to sort by First Name and Last Name, but now I'm being prompted to include other options such as Job Title and Department.

Is this possible? How can I do it?

I've already attempted to add the department, but when I save, I encounter this error:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/101891060/c8690150-26a4-4a61-801b-bc1cee4a4512)

Thank you in advance,

**Comment by kasperbolarsen** [comment]
Check the managed property in the schema,  it will show if the managed property is sortable/refinable.
If the property you need to sort by isn't sortable you will have to map it to a RefinableString/RefinableDate/RefinableInt and use that in your sort settings. Remember to add it to Selected Properties before adding it in the sort settings

**Reply by adumith** [reply]
Hello Kasper, 

Thank you so much for your help.

Using this property, 

JobTitle
![image](https://github.com/microsoft-search/pnp-modern-search/assets/101891060/febd5856-4b51-4636-a3d1-f66d48597feb)

I have been able to create another one:
tkjobtitle
![image](https://github.com/microsoft-search/pnp-modern-search/assets/101891060/e7f5f60d-cd0d-4b68-80f9-6d2402fb3976)

But, if you look at the properties you will notice that the new one doesn't have the refine and sort setting's checked, so I'm not sure if I'm in the right direction. If so, I was wondering if you can guide me to go to the next steps.

Thank you in advance.

**Comment by adumith** [comment]
I got it sir....

After creating the property I was able to add it in the sorting section.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/101891060/e5969cb6-7fd6-469d-8d97-b5d152024020)

Thank you so much.

How ever, I'm facing another issue. The vertical filter is no longer available 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/101891060/27b887b0-d188-4aed-b50b-5f2208dc9f3e)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/101891060/c6351b8c-1581-4dd5-aa5e-65bc640e7dfb)

Any idea how to troubleshoot this?

Thank you again.

**Reply by kasperbolarsen** [reply]
I am not sure what you mean by "created", I would have expected you to add a mapping to one of the existing RefinableStrings, which is also the one you should use as a refiner
